workshops/README.md

```diff
@@ -24,42 +30,50 @@ You'll start off by building your first website and will eventually make your wa
 4. [Geometric Pattern][geometric_pattern]: _Create rad shapes and patterns through code._
 5. [Dodge][dodge]: _Build your first browser game._
 6. [Platformer][platformer]: _Build on what you learned in Dodge to create your second, more complex game._
-7. [Chat][chat]: _Use a database to create a live chat website._
-8. [Collaborative Sketch][collaborative_sketch]: _Use your new database skills to create a real-time collaborative drawing app._
+7. [Collaborative Sketch][collaborative_sketch]: _Use your new database skills to create a real-time collaborative drawing app._
```

Comment:
@zachlatta Chat hasn't been revised by @kyleemile and I yet, so it's in sandbox until we run through it together.